scripts/accessibility-tests.sh

```diff
@@ -1,6 +1,17 @@
 #!/usr/bin/env bash
 set -e
 
+###############################################################################
+#
+# Usage:
+#   To run just tests, without pa11ycrawler:
+#       ./scripts/accessibility-tests.sh
+#
+#   To run tests, followed by pa11ycrawler:
+#       RUN_CRAWLER=1 ./scripts/accessibility-tests.sh
```

Comment:
nit: if you change the envar name to RUN_PA11Y_CRAWLER as suggested below, pls change here too.